_posts/2022-05-31-back-but-small-issues.md

```diff
@@ -0,0 +1,12 @@
+---
+title: MOC Services Available for Use Except ocp-prod Cluster
+state: active
+severity: 1
+---
+
+The Mass Open Cloud services are available for use:
+- We are still working through some minor issues.
```

Comment:
Again, because we want to keep salient information in the first paragraph, I would consider:

> Most Mass Open Cloud services are available for regular use. We are continuing to investigate some issues with the `ocp-prod` OpenShift cluster. If you encounter any unexpected behavior, please contact us via our [ticketing system](https://osticket.massopen.cloud).

Reply:
@larsks  Just so I understand, even with no space between lines 7 and 8 the bulleting will create a new paragraph?